op-deployer/pkg/deployer/bootstrap/consolidated.go

```diff
@@ -0,0 +1,263 @@
+	"github.com/ethereum-optimism/optimism/op-service/jsonutil"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/urfave/cli/v2"
+)
+
+// ConsolidatedOutput combines the outputs from Superchain and Implementations bootstrap
+type ConsolidatedOutput struct {
+	Superchain      opcm.DeploySuperchainOutput      `json:"superchain"`
+	Implementations opcm.DeployImplementationsOutput `json:"implementations"`
+}
+
+type ConsolidatedConfig struct {
+	L1RPCUrl                        string
+	PrivateKey                      string
+	Logger                          log.Logger
+	ArtifactsLocator                *artifacts.Locator
+	Outfile                         string
+	MIPSVersion                     int
+	WithdrawalDelaySeconds          uint64
+	MinProposalSizeBytes            uint64
+	ChallengePeriodSeconds          uint64
+	ProofMaturityDelaySeconds       uint64
+	DisputeGameFinalityDelaySeconds uint64
+	SuperchainProxyAdminOwner       common.Address
+	ProtocolVersionsOwner           common.Address
+	Guardian                        common.Address
+	Paused                          bool
+	RequiredProtocolVersion         string
+	RecommendedProtocolVersion      string
+	UseInterop                      bool
+	CacheDir                        string
+
+	privateKeyECDSA *ecdsa.PrivateKey
+}
+
+func (c *ConsolidatedConfig) Parse() error {
+	if c.L1RPCUrl == "" {
+		return fmt.Errorf("L1RPCUrl must be specified")
+	}
+
+	if c.PrivateKey == "" {
+		return fmt.Errorf("PrivateKey must be specified")
+	}
+
+	var err error
+	c.privateKeyECDSA, err = crypto.HexToECDSA(strings.TrimPrefix(c.PrivateKey, "0x"))
+	if err != nil {
+		return fmt.Errorf("failed to parse private key: %w", err)
+	}
+
+	if c.ArtifactsLocator == nil {
+		c.ArtifactsLocator = artifacts.DefaultL1ContractsLocator
+	}
+
+	if c.MIPSVersion == 0 {
+		c.MIPSVersion = int(standard.MIPSVersion)
+	}
+
+	if c.SuperchainProxyAdminOwner == (common.Address{}) {
+		return fmt.Errorf("SuperchainProxyAdminOwner must be specified")
+	}
+
+	if c.ProtocolVersionsOwner == (common.Address{}) {
+		return fmt.Errorf("ProtocolVersionsOwner must be specified")
+	}
+
+	if c.Guardian == (common.Address{}) {
+		return fmt.Errorf("Guardian must be specified")
+	}
+
+	return nil
+}
+
+var ConsolidatedFlags = []cli.Flag{
+	deployer.L1RPCURLFlag,
+	deployer.PrivateKeyFlag,
+	OutfileFlag,
+	deployer.ArtifactsLocatorFlag,
+	MIPSVersionFlag,
+	WithdrawalDelaySecondsFlag,
+	MinProposalSizeBytesFlag,
+	ChallengePeriodSecondsFlag,
+	ProofMaturityDelaySecondsFlag,
+	DisputeGameFinalityDelaySecondsFlag,
+	SuperchainProxyAdminOwnerFlag,
+	ProtocolVersionsOwnerFlag,
+	GuardianFlag,
+	PausedFlag,
+	RequiredProtocolVersionFlag,
+	RecommendedProtocolVersionFlag,
+	UseInteropFlag,
+}
+
+func ConsolidatedCLI(ctx *cli.Context) error {
```

Comment:
The cli command you are adding is `bootstrap full` but the code here calls it `consolidated` instead. Let's align the two and use `full` for this file name and its functions and in the documentation